identity_base_config/recipes/default.rb

```diff
@@ -64,19 +64,16 @@
   mode '0755'
 end
 
-cookbook_file '/usr/local/bin/id-git' do
-  source 'id-git'
-  owner 'root'
-  group 'root'
-  mode '0755'
-end
-
-cookbook_file '/usr/local/bin/id-rails-console' do
-  source 'id-rails-console'
-  owner 'root'
-  group 'root'
-  mode '0755'
-end
+template "/usr/local/bin/id-rails-console" do
+    source "id-rails-console.erb"
+    owner 'root'
+    group 'root'
+    mode '0755'
+    variables ({
+      :webhook => ConfigLoader.load_config(node, "slackwebhook"),
```

Comment:
We should drop the credentials into a separate file, not a template, so we can make the perms non world readable while still having id-rails-console itself world readable.

Reply:
Ah hah, that methodology is what I was missing; will correct.